drivers/bus/acpi/main.c

```diff
@@ -234,8 +228,6 @@ ACPIDispatchDeviceControl(
     ULONG Caps = 0;
     HANDLE ThreadHandle;
 
-    PAGED_CODE ();
```

Comment:
The function can also be called in non-paged mode? This seems unusual...

Reply:
IOCTL dispatch routines are generally not paged. See also https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/acpiioct/ni-acpiioct-ioctl_acpi_async_eval_method

Reply:
Ah, you are pointing to this:
> IOCTL_ACPI_ASYNC_EVAL_METHOD _**can be used only at IRQL <= DISPATCH_LEVEL.**_

otherwise if it was IRQL strictly lower than dispatch, then it would have certainly been PAGED_CODE.